Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -80,7 +86,7 @@ class ExceptionHandler
   }
 
 private:
-  std::exception_ptr m_FirstCaughtException;
+  std::exception_ptr m_FirstCaughtException{ nullptr };
```

Comment:
I guess this is just a matter of style, right? An `std::exception_ptr` is already _null_ by default, not really necessary to initialize to `nullptr`: 
> A default-constructed std::exception_ptr is a null pointer; it does not point to an exception object. 

https://en.cppreference.com/w/cpp/error/exception_ptr

Reply:
done